src/rules/keywords.md

```diff
@@ -0,0 +1,14 @@
+# Keywords
+
+## Concentration
+
+Some spells and other effects require you to maintain concentration in order to keep them active. If you lose concentration, such an effect ends.
+
+If a spell must be maintained with concentration, that fact appears in its Duration entry, and the spell specifies how long you can concentrate on it. You can end concentration on any such effect at any time (no action required).
```

Comment:
The last sentence should probably go somewhere else, since it's not like you can only drop conc on spells